bundle/edu.gemini.sp.vcs/src/main/scala/edu/gemini/sp/vcs2/Vcs.scala

```diff
@@ -66,40 +78,48 @@ class Vcs(user: VcsAction[Set[Principal]], server: VcsServer, service: Peer => V
     def filter(eval: MergeEval): Boolean = eval.localUpdate
 
     def update(f: ISPFactory, p: ISPProgram, eval: MergeEval): VcsAction[Unit] =
-      eval.plan.merge(f, p)
+      checkCancel(cancelled) >> eval.plan.merge(f, p)
```

Comment:
This documentation maybe useful
http://eed3si9n.com/learning-scalaz/scalaz-cheatsheet.html

`>>` appears there thou is not that clear what it does from the exmaple